### ISSUE TITLE
allow to work with mutlithreaded OpenBLAS

### ISSUE THREAD

**Issue by ClementPernet** [linbox-team/fflas-ffpack#42]
So far we used to require to link to a single threaded BLAS.
This will cause pb for the debianization, as openblas is packaged in its multithreaded version.

SageMath also ships a multithreaded OpenBLAS.

From ​https://github.com/xianyi/OpenBLAS/wiki/faq 
we can force OpenBLAS to run sequentially using `openblas_set_num_threads(1)`.
Let's do it!

**Comment by ClementPernet** [comment]
Fixed in #44

**Comment by dlesnoff** [comment]
With fflas-ffpack 2.4.3 and this compiling option, I get:
> g++ -Wall -Wextra -Wpointer-arith -O3 -march=native -o fflas-float_matmul fflas_matmul.cc -lblas -lopenblas -lgivaro -I/local
```raw
/lesnoff/local/include -fopenmp -D __MULTITHREADING__ -D __SGEMM__
fflas_matmul.cc: In function ‘int main(int, char**)’:
fflas_matmul.cc:44:3: error: ‘openblas_set_num_threads’ was not declared in this scope; did you mean ‘flint_set_num_threads’?
   44 |   openblas_set_num_threads(1);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~
      |   flint_set_num_threads
```

Here is a minimal code example:
```cpp
#include<fflas-ffpack/fflas-ffpack-config.h>
#include<givaro/modular-balanced.h>
#include<fflas-ffpack/fflas/fflas.h>
#include<fflas-ffpack/utils/fflas_io.h>

#include<iostream>
#include<fstream>
#include<chrono>

#ifndef __SGEMM__
typedef double Floats;
#define CBLAS_GEMM cblas_dgemm
#else
typedef float Floats;
#define CBLAS_GEMM cblas_sgemm
#endif

using namespace FFLAS;
using namespace Givaro;

int main()
{
  // Otherwise BLAS multithreading conflict with FFLAS multithreading
  openblas_set_num_threads(1);
  return 0;
}
```

I thought that the problem was that it missed the `#include<cblas.h>` header but apparently I get conflicting declarations with this.

Has this issue been fixed with fflas-ffpack v2.5.0? I would like to change the fflas-ffpack version locally in my project, but I do not see how I could change the version without a compiler flag.

Reopening this issue felt more suited than opening a new one but I can move this issue to another one if desired.
